src/coreclr/src/jit/lowerxarch.cpp

```diff
@@ -218,6 +218,16 @@ void Lowering::LowerBlockStore(GenTreeBlk* blkNode)
 #endif
         }
     }
+    else if (src->OperIsHWIntrinsic())
```

Comment:
@CarolEidt I think I need to implement a similar logic on Arm64 to avoid store-reload of returned SIMD values for `ldp`/`ldnp` as in the following example:
```asm
        2C404410          ldnp    s16, s17, [x0]
        FD000FB0          str     d16, [fp,#24]
        FD0013B1          str     d17, [fp,#32]
        FD400FA8          ldr     d8, [fp,#24]
        FD4013A9          ldr     d9, [fp,#32]
```

Reply:
Yes, I believe that's correct. However, this path is only for the case where you wind up with a `STORE_BLK`, as opposed to a `STORE_LCL_VAR`. If the lhs is a multi-reg lclVar, you should have the latter.